chef_master/source/perform_compliance_scan.rst

```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
 
 .. end_tag
 
+.. tag compliance_beta
+
+.. important:: The new compliance functionality described in this topic is currently in Beta. To enable compliance reporting in the Chef Automate UI, navigate to the **Nodes** tab, make sure your cursor is not in any text box or field, and type ``beta``. A new ``Compliance`` tab should appear in the top-level menu of the UI. Note: Enabling the ``Compliance`` tab will allow you to view only new compliance scan data, not historical data.
```

Comment:
As of Chef Automate 1.5.46, the integrated compliance functionality is GA, so we shouldn't need to add this note back in.